src/sizeMapping.js

```diff
@@ -65,22 +65,20 @@ export function resolveStatus({labels = [], labelAll = false, activeLabels = []}
   let maps = evaluateSizeConfig(configs);
 
   if (!isPlainObject(mediaTypes)) {
-    mediaTypes = {};
+    // add support for deprecated adUnit.sizes by creating correct banner mediaTypes if they don't already exist
+    if (sizes) {
```

Comment:
Should this check if mediaTypes already exists, as stated in description?

I'm testing this fix out, but I still see mediaTypes.banner.sizes set when I'm only configuring an instream video mediaType.

Reply:
My mistake, I was testing a prebid version without this fix. With the fix, I can confirm that it is working as expected and no longer setting mediaType.banner if only a video unit.

Thanks for the fix!